docs/docs/guides/manage/self-hosted/configure/docker-compose.yaml

```diff
@@ -1,53 +1,55 @@
-version: '3.8'
```

Comment:
Are we sure we want to change single to double quotes?

Reply:
My vscode linter did that 🤷‍♂️😬

Reply:
I think it doesn't make a big difference, but [escaping characters is more similar with other languages when using double quotes](http://yaml.org/spec/1.2-old/spec.html#id2776092). Therefore, double quotes are slightly better, IMO.

Reply:
in that case i think we should set up a linter for yaml stuff.

I am fine with using doubles if we stick to it 😁

Reply:
Added formatting for all **.md files and everything in the /docs folder.